### ISSUE TITLE
Option to specify delimiter between chapter and Vers(es) in Bible references

### ISSUE THREAD

**Issue by reckel-jm** [AndBible/and-bible#3281]
**Is your feature request related to a problem? Please describe.**
In some languages as German, a Bible reference usually has a different formation. Instead of an :, a simple komata is used to separate the chapter from the verses.

For example: 
English: Matthew 1:1
German: Matthäus 1,1

However as of now, And Bible will always use the English format (Matthäus 1:1)–even so when the language is different.

**Describe the solution you'd like**
An option in the setting which allows to specify the desired format of Bible verse references.

**Describe alternatives you've considered**
It could also be hard coded for every single language.

**Additional context**
It would help to improve a native feeling.

**Comment by tuomas2** [comment]
Probably delimeter should be set correctly based on locale.

**Comment by tuomas2** [comment]
Is it consistently comma in all german biblical texts?

**Comment by reckel-jm** [comment]
Yes, except of some groups like Jehovas Withnesses which use the colon (because they come from the US), the comma is absolutely the normal delimiter for German bible references.

**Comment by tuomas2** [comment]
Asked ChatGPT which languages do not use colon.

> Certainly! Several languages use different conventions for delimiting chapters and verses in Bible references. Here are a few examples:
> 
> 1. **French**:
>    - Uses a comma: Genèse 1,1
> 
> 2. **Italian**:
>    - Uses a comma: Genesi 1,1
> 
> 3. **Spanish**:
>    - Uses a comma: Génesis 1,1
> 
> 4. **Portuguese**:
>    - Uses a comma: Gênesis 1,1
> 
> 5. **Dutch**:
>    - Uses a colon, similar to English, but in some contexts, a comma may also be used: Genesis 1:1 or Genesis 1,1
> 
> 6. **Finnish**:
>    - Uses a comma: 1. Mooseksen kirja 1,1
> 
> 7. **Swedish**:
>    - Uses a comma: Första Moseboken 1,1
> 
> These examples illustrate that while many languages follow a convention similar to English with a colon, there are notable exceptions, particularly among Romance languages and some other European languages, which prefer a comma to separate chapter and verse.

**Comment by tuomas2** [comment]
Though in his answer, Finnish is wrong. We use colon commonly .

**Comment by tuomas2** [comment]
ChatGPT later added also Czech to his answer

**Comment by reckel-jm** [comment]
Yes this is quite possible that other languages do so as well.
